chapters/chap03-sec02.tex

```diff
@@ -109,7 +109,7 @@ \subsection{An efficient instantiation of \mimc{} primitives}\label{instantiatio
     To increase the security of the $\mathsf{\mkhash}$, different round constants for each level of the Merkle tree could be used.
 \end{remark}
 
-The value of $p$ is set to $\rCURVE$ (specific to the curve being used). Thereby, for input values $m_0$ and $m_1$, we define $\mkhash : \FFx{\rCURVE} \times \FFx{\rCURVE} \to \FFx{\rCURVE}$ by
+We define $\mkhash$ to be $\mimcMPPrime{}$ over the $\FFx{\rCURVE}$. Thereby, for input values $m_0$ and $m_1$, $\mkhash : \FFx{\rCURVE} \times \FFx{\rCURVE} \to \FFx{\rCURVE}$ is defined by
```

Comment:
I'd rephrase to either:
`We define $\mkhash$ to be $\mimcMPPrime{\rCURVE}$` or `We define $\mkhash$ to be $\mimcMPPrime{}$ over the field $\FFx{\rCURVE}$` or `We define $\mkhash$ to be $\mimcMPPrime{}$ over $\FFx{\rCURVE}$`

Reply:
removed the extra `the`.  thanks.